scipp_widgets/validators.py

```diff
@@ -47,6 +48,12 @@ def __call__(self, input):
 scipp_object_validator = TypeValidator(scipp_object)
```

Comment:
Should this be `ScippObjectValidator`? I know it's refering to a constructed object, but it's maybe a little weird to `import scipp_object_validator` when everything else is `import TypeValidator`?

In addition, do we have a better name for describing `Variable`, `DataArray` etc..? I have "scipp object" in the plotting code but never been super happy with it? Does anyone have a better idea?

Reply:
I can't think of a better name than `scipp_object` currently but will continue to ruminate.